index.js

```diff
@@ -117,19 +116,9 @@ function checkOptionUsage(longOption, optionValue, options,
  */
 function storeOption(longOption, optionValue, options, values) {
   if (longOption === '__proto__') {
-    return;
+    return; // No. Just no.
```

Comment:
i'm not actually sure we should be returning here - we can just set a data property of the string "__proto__", no?

Reply:
My concern is it is fine in the `values` we return, but is it a potential foot-gun when the author copies it out into some full object? Or does who-knows-what? We are going to great lengths to make this as safe as we can, why would we allow it?

(I might have missed the point though. Not sure what you meant by `string "proto"`.)

Reply:
It would only be a footgun if a user is relying on the (deletable) `Object.prototype.__proto__` accessor. Otherwise, it'd just be a string property.

If we want to disallow it, why not throw in strict mode?

Reply:
Existing behaviour. I care enough about it to add a comment showing it is deliberate. But not to do work on it in this PR. 😄

Reply:
lol totally fine, we can look at it later.